src/Chainweb/Miner/RestAPI/Server.hs

```diff
@@ -135,7 +137,9 @@ updatesHandler :: CutDb cas -> ChainBytes -> Tagged Handler Application
 updatesHandler cdb (ChainBytes cbytes) = Tagged $ \req respond -> do
     cid <- runGet decodeChainId cbytes
     cv  <- _cut cdb >>= newIORef
-    eventSourceAppIO (go cid cv) req respond
+    x <- randomRIO @Double (0.9, 1.1)
+    timer <- registerDelay (round $ 1000000 * 240 * x)
+    eventSourceAppIO (go timer cid cv) req respond
```

Comment:
Can you put comments here to explain the units?

Reply:
Good point. Also, I wonder if the use of the STM timer should be avoided. It's probably more efficient to just get the current time after each event. An even simpler solution would terminate the stream based on the number of events instead of time.